test/index.js

```diff
@@ -9,6 +9,7 @@ var grunt = require('grunt'),
 test('Grunt-localizr', function (t) {
```

Comment:
could you pr those in then ? we can merge mine and then yours as well against that.